src/components/a-icon/index.scss

```diff
@@ -1,11 +1,16 @@
 a-icon__root {}
 
-.a-icon {
+.a-icon .a-icon__svg {
```

Comment:
This CSS specificity seems to be to high

Reply:
seems or is? might i ask you for more details. Can we live with it or in your opinion is better to be less specific in this contest?

I generally worry about specificity as for a consumer for the patterns library is harder to override classes if they need to.

Reply:
@MarekLacoAXA if `.a-icon__svg {}` can live without the need to increase specificity, please do so as @AndyOGo suggests

Reply:
I changed it.  So we have to use a specific icon class "modifier" to pass down that sits on the svg element.